example/cumulus-tf/python-reference-task.tf

```diff
@@ -6,7 +6,7 @@ resource "aws_lambda_function" "python_reference_task" {
   role             = module.cumulus.lambda_processing_role_arn
   runtime          = "python3.7"
   timeout          = 300
-  memory_size      = 256
+  memory_size      = 1556
```

Comment:
So again, the idea here is that the extra memory improves performance and reduces execution time?

Reply:
More or less.  It gives it more horsepower, and the improvement is linear. 

Upside - tests run seconds faster. 
Downside - screwing up might cost a lot more.